data/vocab/scholar.ttl

```diff
@@ -847,7 +172,245 @@ scholar:Agency a rdfs:Class ;
+    rdfs:subClassOf bf:Work .
+
+scholar:PressRelease a rdfs:Class ;
+    rdfs:label "Press Release"@en ;
+    rdfs:comment "An official statement issued to newspapers and other news media for the purpose of providing information, an official announcement, or making a public statement."@en ;
+    rdfs:subClassOf bf:Work .
+
+scholar:Proposal a rdfs:Class ;
+    rdfs:label "Proposal"@en ;
+    rdfs:comment "A plan or suggestion, especially a formal or written one, put forward for consideration by others."@en ;
+    rdfs:subClassOf bf:Work .
+
+scholar:Software a rdfs:Class ;
+    rdfs:label "Software"@en ;
+    rdfs:comment "A program or set of programs that instruct a computer to perform specific tasks."@en ;
+    rdfs:subClassOf bf:Work .
+
+scholar:SpecialReport a rdfs:Class ;
+    rdfs:label "Special Report"@en ;
+    rdfs:comment "A detailed and comprehensive analysis on a specific topic or event."@en ;
+    rdfs:subClassOf scholar:Report .
+
+scholar:Summary a rdfs:Class ;
+    rdfs:label "Summary"@en ;
+    rdfs:comment "A brief statement or account covering the main points of a larger work or article."@en ;
+    rdfs:subClassOf bf:Work .
+
+scholar:SlideShow a rdfs:Class ;
+    rdfs:label "Slide Show"@en ;
+    rdfs:comment "A presentation of a series of still images on a projection screen or electronic display device, typically in a prearranged sequence."@en ;
+    rdfs:subClassOf bf:Work .
+
+scholar:Publisher a rdfs:Class ;
+    rdfs:label "Publisher"@en ;
+    rdfs:comment "An entity responsible for making the resource available, including companies, organizations, or individuals."@en ;
+    rdfs:subClassOf bf:ProviderAgent .
+
+scholar:Policy a rdfs:Class ;
+    rdfs:label "Policy"@en ;
+    rdfs:comment "A course or principle of action adopted or proposed by an organization or individual."@en ;
+    rdfs:subClassOf bf:Resource .
+
+scholar:OpenAccessPolicy a rdfs:Class ;
+    rdfs:label "Open Access Policy"@en ;
+    rdfs:comment "A policy that promotes open access to scholarly works and resources."@en ;
+    rdfs:subClassOf scholar:Policy .
+
+scholar:DepositAccess a rdfs:Class ;
+    rdfs:label "Deposit Access"@en ;
+    rdfs:comment "Policies or guidelines related to the depositing of scholarly works into repositories or archives."@en ;
+    rdfs:subClassOf scholar:Policy .
+
+scholar:ActionEvent a rdfs:Class ;
+    rdfs:label "Action Event"@en ;
+    rdfs:comment "Represents an action or event within the lifecycle of a scholarly activity, such as the awarding of a grant, submission of a paper, or any other significant occurrence that has implications for the research or academic work."@en ;
+    rdfs:subClassOf bf:Event .
+
+## BF Scholar Identifier Classes
+
+scholar:EbscoID a rdfs:Class ;
+	rdfs:label "EBSCO ID"@en ;
+	rdfs:comment "An EBSCO ID associated with an article, person, or organization."@en ;
+    rdfs:subClassOf bf:Identifier .
```

Comment:
EBSCO has many IDs for things. If this is necessary, this should be more specific.